frontend-react/src/hooks/api/messages/UseTestMessageResult/UseTestMessageResult.ts

```diff
@@ -0,0 +1,75 @@
+import { useQuery } from "@tanstack/react-query";
```

Comment:
Should be using `useSuspenseQuery`

Reply:
hmm ok, then we're due for potentially refactoring our code to move enabled logic out of the hooks and into the components